docs/source/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cython>=0.29.21
```

Comment:
This file can get dropped.

`nox -s build_docs` handles things locally.
The RTD config file uses pip to install necessary packages for documentation.